src/WebJobs.Extensions.ApiHub/ApiHubFile.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Azure.ApiHub;
+using Microsoft.Azure.WebJobs.Extensions.ApiHub.Common;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub
+{
+    class ApiHubFile : IFileStreamProvider
```

Comment:
Also, please include some summary doc blocks on all these new types - that would help me and others understand them :)